From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 10 Dec 2016 04:34:55 +0000
Subject: ubi: Avoid ABI change in 3.16.37
Forwarded: not-needed

struct ubi_device was changed by commit ec4dac52d2b5 ("UBI: Fix static
volume checks when Fastmap is used").  It is referred to indirectly
from struct ubi_volume, but its definition is private to ubi and is
not included by any other module.  So hide the new field from
genksyms; don't even bother to reorder it.

---
--- a/drivers/mtd/ubi/ubi.h
+++ b/drivers/mtd/ubi/ubi.h
@@ -532,7 +532,9 @@ struct ubi_device {
 	void *fm_buf;
 	size_t fm_size;
 	struct work_struct fm_work;
+#ifndef __GENKSYMS__
 	int fast_attach;
+#endif
 
 	/* Wear-leveling sub-system's stuff */
 	struct rb_root used;
